src/components/signup/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
change name of folder. You can move this file and related scc to payment folder and rename the file to FirstSignUp and respectively css file